src/Salt/UserBundle/Controller/UserController.php

```diff
@@ -33,7 +33,7 @@ class UserController extends Controller
      * Lists all user entities.
      *
      * @Route("/", name="admin_user_index")
-     * @Method("GET")
+     * @Method({"GET"})
```

Comment:
why is this curly braces added? I believe we don't need to modify this file for this approach